tests/test_push_cgw.py

```diff
@@ -55,3 +55,56 @@ def test_cgw_operations_exception():
         push_cgw.cgw_operations()
 
     assert push_cgw.rollback_cgw_operation.called is True
+
+
+@mock.patch("pubtools._content_gateway.push_cgw.PushCGW")
+@mock.patch(
+    "pubtools._content_gateway.push_cgw.argparse.ArgumentParser.parse_args",
+    return_value=argparse.Namespace(
+        CGW_hostname="http://fake_host_name/test",
+        CGW_username="test_username",
+        CGW_password=None,
+        CGW_filepath=yaml_file_path,
+    ),
+)
+def test_main_with_env_var_password(mock_args, mock_push_cgw):
+    os.environ["CGW_PASSWORD"] = "test_password_from_env"
```

Comment:
you can define pytest fixture like:
```
def mock_cgw_password():
    os.environ['CGW_PASSWORD'] = "test"
    yield 
    del os.environ['CGW_PASSWORD']
```
and then add it to test functions like:
```
test_main_with_env_var_password(mock_args, mock_push_cgw,mock_cgw_password):
```

Reply:
done. thanks